cpg_pipes/providers/cpg/inputs.py

```diff
@@ -162,22 +164,23 @@ def populate_alignment_inputs(self, cohort: Cohort) -> None:
         found_seqs_by_sid = defaultdict(list)
         for found_seq in found_seqs:
             found_seqs_by_sid[found_seq['sample_id']].append(found_seq)
-        
+
         # Checking missing sequences
         if sample_wo_seq := [
             s for s in cohort.get_samples() if s.id not in found_seqs_by_sid
         ]:
             msg = f'No sequencing data found for samples:\n'
+            ds_sample_count = {ds_name: len(list(ds_samples)) for ds_name, ds_samples in groupby(cohort.get_samples(), key=lambda s: s.dataset.name)}
             for ds, samples in groupby(sample_wo_seq, key=lambda s: s.dataset.name):
                 msg += (
-                    f'\t{ds}, {len(list(samples))} samples: '
+                    f'\t{ds}, {len(list(samples))}/{ds_sample_count.get(ds)} samples: '
                     f'{", ".join([s.id for s in samples])}\n'
                 )
-            if get_config()['workflow'].get('smdb_errors_are_fatal', True):
-                raise InputProviderError(msg)
-            else:
-                msg += '\nContinuing without sequencing data because lenient=true'
-                logger.warning(msg)
+            # if get_config()['workflow'].get('smdb_errors_are_fatal', True):
```

Comment:
Should we remove this flag and remove the commendet lines?

Reply:
I've converted this to a `logger.info` call, removed the commented out lines, rephrased the message to also include the sequencing type + added a comment